tests/rgw/sanity_rgw.py

```diff
@@ -187,14 +187,13 @@ def run(ceph_cluster, **kw):
 
     if run_io_verify:
         log.info("running io verify script")
-        verify_out, err = exec_from.exec_command(
+        exec_from.exec_command(
```

Comment:
@hmaheswa , not sure if these changes are specific to bucket notification suite. If they are not, could we have a separate PR addressing the issue and fix

Reply:
@mkasturi18 , reverted the changes back, will have them as a separate PR